### PR TITLE
[CELEBORN-193][FOLLOWUP] Reduce unnecessary binary files.

### PR DESCRIPTION
**Pull request by FMX**

### CODE REVIEW

**Review by waitinfuture — APPROVED** [review]

File: build/make-distribution.sh [hunk 1]
```diff
@@ -220,5 +220,5 @@ TARDIR_NAME="apache-celeborn-$VERSION-$NAME"
 TARDIR="$PROJECT_DIR/$TARDIR_NAME"
 rm -rf "$TARDIR"
 cp -R "$DIST_DIR" "$TARDIR"
-tar czf "apache-celeborn-$VERSION-$NAME.tgz" -C "$PROJECT_DIR" "$TARDIR_NAME"
+tar -cz --no-xattrs --disable-copyfile --exclude='.DS_Store' -f "apache-celeborn-$VERSION-$NAME.tgz" -C "$PROJECT_DIR" "$TARDIR_NAME"
```
Comment (pan3793) [inline build/make-distribution.sh:223]: what's "--disable-copyfile" for? I can not find it in `man tar`

Comment (pan3793) [reply build/make-distribution.sh:223]: do we need `--exclude='.DS_Store'`? it only exists if you opened the folder using `Finder`, I think we don't need that.

**Comment by waitinfuture** [comment]
I've tested on CentOS and things goes well.

**Review by pan3793 — APPROVED** [review]
The change lgtm, even we have not found the exact condition cause this problem.

**Pull request merged**